containers/datalab/content/.bashrc

```diff
@@ -1,8 +1,8 @@
 # ~/.bashrc: executed by bash(1) for non-login shells.
 
 cd /content
-STARTCOLOR='\e[1;32m';
-ENDCOLOR="\e[0m"
+STARTCOLOR='\[\e[1;32m\]';
+ENDCOLOR="\[\e[0m\]"
 export PS1="$STARTCOLOR\u# \w> $ENDCOLOR"
```

Comment:
Just cosmetic. I think I'll take it out along with the username, since our instances are meant to support a single user anyway, and just keep it simple with the CWD only.